### PR TITLE
Fixed: SemiPublic => SemiPrivate

### PR DESCRIPTION
**Pull request by bakerboy448**
#### Database Migration
NO

#### Description
Fixed incorrect enum and related

#### Todos
- Tests
- [X] Translation Keys (./src/NzbDrone.Core/Localization/Core/en.json)
-  [Wiki Updates](https://wiki.servarr.com)

#### Issues Fixed or Closed by this PR

* Fixes #XXXX

### CODE REVIEW

File: src/NzbDrone.Core/Indexers/Definitions/Cardigann/Cardigann.cs [hunk 1]
```diff
@@ -132,7 +132,7 @@ private IndexerDefinition GetDefinition(CardigannMetaDefinition definition)
                 {
                     "private" => IndexerPrivacy.Private,
                     "public" => IndexerPrivacy.Public,
-                    _ => IndexerPrivacy.SemiPublic
+                    _ => IndexerPrivacy.SemiPrivate
```
Comment (bakerboy448) [inline src/NzbDrone.Core/Indexers/Definitions/Cardigann/Cardigann.cs:135]: YAML will be `type: semi-private`  should we not handle this explictly?

Comment (Qstick) [reply src/NzbDrone.Core/Indexers/Definitions/Cardigann/Cardigann.cs:135]: A C# switch needs a default. so it's fine as it is

File: frontend/src/Indexer/Add/AddIndexerModalContent.js [hunk 2]
```diff
@@ -62,8 +62,8 @@ const privacyLevels = [
     value: translate('Private')
   },
   {
-    key: 'semiPublic',
-    value: translate('SemiPublic')
+    key: 'SemiPrivate',
```
Comment (Qstick) [inline frontend/src/Indexer/Add/AddIndexerModalContent.js:65]: enum comes from the backend with first char lower.

**Review by Qstick — CHANGES_REQUESTED** [review]
One nitpick

File: frontend/src/Indexer/Add/AddIndexerModalContent.js (hunk 2 above)
Comment (bakerboy448) [reply frontend/src/Indexer/Add/AddIndexerModalContent.js]: squished

**Review by Qstick — APPROVED** [review]

**Pull request merged**